src/main/resources/changelog.db/changelog-init.xml

```diff
@@ -0,0 +1,20 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="create_table_messge_info" author="Leonid Baidak">
+
+        <createTable tableName="mail_message_info">
+            <column name="id" type="bigint" autoIncrement="false">
```

Comment:
Why shouldn't it be auto-incremented?

Reply:
For my point of view it's not a responsibility of database - it's responsibility of data model, cause in this service we use annotation @javax.persistence.GeneratedValue for id field in the entity, to auto-generate id.

Reply:
`@GeneratedValue` doesn't declare who is responsible for ID generation. If it wasn't a responsibility of database, Postgres team wouldn't implement that. I'm ok if you use default ID generation (without IDENTITY strategy), but my point is next: if DB can work with your DDL - let it do so, otherwise, put the responsibility on the JPA.

Besides that, your schema won't work. To generate an ID with a default `@GeneratedValue` hibernate needs a `sequence`, which you doesn't create. And there is one more point - by default Hibernate shares that sequence across all the tables, so they will share IDs. If you want to use `sequence`, you need to create it.